Dockerfile

```diff
@@ -43,8 +43,10 @@ RUN apk add --no-cache \
   vo-amrwbenc-dev vo-amrwbenc-static \
   snappy snappy-dev snappy-static \
   xxd \
-  xz-dev xz-static
-
+  xz-dev xz-static \
+  asciidoc gdk-pixbuf-dev gflags-dev gtest-dev \
+  openexr-dev samurai curl bash
```

Comment:
samurai seems to be ninja replacement? can ninja be used as it's already installed? maybe check the other packages also if they are required or can be skipped somehow, skip docs etc, but it's not super important if it does not affect the resulting docker image

Reply:
It seems to be required somehow by meson/ninja
```
World updated, but the following packages are not removed due to:
  samurai: ninja meson harfbuzz-dev libxml2-static
```
while other packaged are not actually required anymore (without asciidoc it will disable documentation automatically and tests are disabled), with only curl as an expection as it is used in the script from libjxl.